hack/make/.build-rpm/docker-engine.spec

```diff
@@ -0,0 +1,168 @@
+Name: docker-engine
+Version: %{_version}
+Release: %{_release}%{?dist}
+Summary: The open-source application container engine
+
+License: ASL 2.0
+Source: %{name}.tar.gz
+
+URL: https://dockerproject.com
+Vendor: Docker
+Packager: Docker <support@docker.com>
+
+# is_systemd conditional
+%if 0%{?fedora} >= 21 || 0%{?centos} >= 7 || 0%{?rhel} >= 7
+%global is_systemd 1
+%endif
+
+# required packages for build
+# most are already in the container (see contrib/builder/rpm/generate.sh)
+# only require systemd on those systems
+%if 0%{?is_systemd}
+BuildRequires: pkgconfig(systemd)
+Requires: systemd-units
+%else
+Requires(post): chkconfig
+Requires(preun): chkconfig
+# This is for /sbin/service
+Requires(preun): initscripts
+%endif
+
+# required packages on install
+Requires: /bin/sh
+Requires: iptables
+Requires: libc.so.6
+Requires: libdevmapper.so.1.02
+Requires: libpthread.so.0
+Requires: libsqlite3.so.0
+Requires: selinux-policy >= 3.13.1-114
+Requires: tar
+Requires: xz
+%if 0%{?fedora} >= 21
+# Resolves: rhbz#1165615
+Requires: device-mapper-libs >= 1.02.90-1
+%endif
+
+# conflicting packages
+Conflicts: docker
+Conflicts: docker-io
+
+%description
+Docker is an open source project to pack, ship and run any application as a
+lightweight container
+ 
+Docker containers are both hardware-agnostic and platform-agnostic. This means
+they can run anywhere, from your laptop to the largest EC2 compute instance and
+everything in between - and they don't require you to use a particular
+language, framework or packaging system. That makes them great building blocks
+for deploying and scaling web apps, databases, and backend services without
+depending on a particular stack or provider.
+
+%prep
+%if 0%{?centos} <= 6
+%setup -n %{name}
+%else
+%autosetup -n %{name}
+%endif
+
+%build
+./hack/make.sh dynbinary
+# ./docs/man/md2man-all.sh runs outside the build container (if at all), since we don't have go-md2man here
+
+%check
+./bundles/%{_origversion}/dynbinary/docker -v
+
+%install
+# install docker
+mkdir -p $RPM_BUILD_ROOT/%{_bindir}
+cp -aT bundles/%{_origversion}/dynbinary/docker-%{_origversion} $RPM_BUILD_ROOT/%{_bindir}/docker
+mkdir -p $RPM_BUILD_ROOT/usr/libexec/docker
+cp -aT bundles/%{_origversion}/dynbinary/dockerinit-%{_origversion} $RPM_BUILD_ROOT/usr/libexec/docker/dockerinit
+
+# copy udev rules
+mkdir -p $RPM_BUILD_ROOT/etc/udev/rules.d
+cp -aT contrib/udev/80-docker.rules $RPM_BUILD_ROOT/etc/udev/rules.d/80-docker.rules
+
+# add init scripts
+mkdir -p $RPM_BUILD_ROOT/%{_unitdir}
+mkdir -p $RPM_BUILD_ROOT/etc/sysconfig
+mkdir -p $RPM_BUILD_ROOT/etc/init.d
+
+cp -aT contrib/init/systemd/docker.service $RPM_BUILD_ROOT/%{_unitdir}/docker.service
+cp -aT contrib/init/systemd/docker.socket $RPM_BUILD_ROOT/%{_unitdir}/docker.socket
+
+cp -aT contrib/init/sysvinit-redhat/docker.sysconfig $RPM_BUILD_ROOT/etc/sysconfig/docker
+cp -aT contrib/init/sysvinit-redhat/docker $RPM_BUILD_ROOT/etc/init.d/docker
```

Comment:
Isn't this supposed to go into `%_initddir`? (https://fedoraproject.org/wiki/Packaging:SysVInitScript, especially https://fedoraproject.org/wiki/Packaging:SysVInitScript#Initscripts_on_the_filesystem)